### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by jasmine-jaksic**
Adding Sven and me as approvers

### CODE REVIEW

**Comment by sdake** [comment]
@jasmine-jaksic I was thinking - you/sven will end up being assigned a boatload of issues in the documentation repository if your in the TLD owners file.  If that is what your want, thats cool, although its a bit daunting to stay on top of :)  If your desire is to be able to approve/deny this file:

https://github.com/istio/istio.github.io/tree/master/content/about/feature-stages

(which would mean you only receive assignments  for that file) it may be better to submit a PR for that specific directory (content/feature-stages).  i can approve that while @geeknoid is on PTO, although iirc he is back today or tomorrow.

Cheers
-steve

**Comment by jasmine-jaksic** [comment]
@sdake There are a lot of doc PRs in flight but not enough approvers who have permissions to merge them. Several team members have been complaining about this especially this week. So I think it doesn't hurt to have back-up approvers.

**Comment by sdake** [comment]
@jasmine-jaksic works for me although I think the PR backlog has been moving pretty fast; not sure what the complaint is about.  I still can't approve this as I'm not in the top level owners file.

Cheers
-steve

/assign @frankbu 
/assign @linsun

**Comment by geeknoid** [comment]
The problem is that updating doc content is more than just changing prose. There are conventions in the docs, many subtle formatting issues, etc.

I'd like to understand the nature of the complaints. Reviews have been done promptly and there is no backlog of PRs to review.

File: OWNERS [hunk 1]
```diff
@@ -3,3 +3,5 @@ approvers:
   - geeknoid
   - linsun
   - frankbu
+  - jasmine-jaksic
+  - smawson
```
Comment (rshriram) [inline OWNERS]: and me??

Comment (andraxylia) [reply OWNERS]: Me too please.

Comment (andraxylia) [reply OWNERS]: The current list of owners is too limited and noticed a slow turnaround.

**Comment by geeknoid** [comment]
​Andra, can you point out when there has been slow turnaround on PRs on
this repo?​

**Comment by andraxylia** [comment]
https://github.com/istio/istio.github.io/pull/1948

2 hours to get the first review on a doc PR is far too long, especially when we are on a rush to finish by 1.0.

**Comment by geeknoid** [comment]
I pushed some updates the OWNERS files, so I'm closing this PR.

@andraxylia 2 hours for a code review is quite reasonable I believe. Beyond that though, the limiting factor is not the owners that can approve the PR, but it is the domain experts that need to look at the specific details.

**Pull request closed without merging**